polls/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link href="https://fonts.googleapis.com/css2?family=Titillium+Web:wght@300&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="../styles/reset.css">
+    <link rel="stylesheet" href="../styles/style.css">
+    <link rel="stylesheet" href="../styles/home.css">
+    <script src="https://cdn.jsdelivr.net/npm/@supabase/supabase-js"></script>
+    <title>Polling Station!</title>
+</head>
+<header>
+    <button class="logout">Logout!</button>
+</header>
+<body>
+    <main>
+        <h1>Welcome to the Polling Station!</h1>
+        <section class="create-poll-area">
+            <form id="poll-input">
+                <label>
+                    Enter a question to poll!
+                    <input name="question-entry">
+                </label>
+                <label>
+                    Enter option 1!
+                    <input name="option1-entry">
+                </label>
+                <label>
+                    Enter option 2!
+                    <input name="option2-entry">
+                </label>
+                <button>Submit</button>
+            </form>
+        </section>
+        <section class="current-poll-area">
+            <div class="current-question">Current Question</div>
+            <div class="buttons-count">
+                <div>
+                    <div class="option1">Option 1</div>
+                    <button class="add-1">+</button>
+                    <button class="subtract-1">-</button>
+                    <div class="count-1"></div>
+                </div>
+                <div>
+                    <div class="option2">Option 2</div>
+                    <button class="add-2">+</button>
+                    <button class="subtract-2">-</button>
+                    <div class="count-2"></div>
+                </div>
```

Comment:
consider giving the divs on lines 41 and 47 a class of options and then give them some margin so they're a little more spaced out in the current poll section